cmd/kubeadm/app/cmd/phases/join/controlplanejoin.go

```diff
@@ -0,0 +1,255 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
2019

Reply:
could please you also rename the file in this folder called `controlplane.go` to `controlplaneprepare.go` as part of this PR?

so that we have the two phases outlined better:
- contolplaneprepare.go
- controlplanejoin.go

this seems better to me.

Reply:
sure